docs/source/physical_robot_installation_guide/index.rst

```diff
@@ -0,0 +1,97 @@
+===============================
+Setting up a new Physical Robot
+===============================
+This tutorial is aimed towards developers, who aim to set up a new robot.
+If you simply want to know how to run a physical robot, check the examples.
+
+-----------
+Preparation
+-----------
+Firstly make sure to have all materials necessary to build the physical robot.
+Models to print the Modules can be found on the following git repo: `<https://github.com/ci-group/revolve-models>`_.
+Other items needed:
+
+* Rasberry Pi (rpi)
+* SD-card for the rpi-OS
+* suitable Battery
+* HAT for the rpi
+* cables
+
+------------------
+Setting up the RPI
+------------------
+To setup the RPI follow the steps below carefully.
+
+#. Flash the SD-card with the RPI-OS. Some Important notes:
+    * Make sure to select a appropriate Keyboard Layout (preferably English(US)).
+
+    * When setting up the WIFI connection, select your Country.
+
+#. Once the SD-card is flashed, put it into the rpi, mounted on the robot and connect to it. If you dont know the IP-address do the following:
```

Comment:
```suggestion
#. Boot the RPi and get command line access. Here are some general tips.
    * You can connect a keyboard and screen.
    * Better is to SSH. This work only if you and the RPi are on the same network. For CI Group, connect it to ThymioNet Wi-Fi.
    * You can find the IP addresses of devices on your network through :code:`sudo nmap -sP <your ip>`.
```

I think we can expect a certain level of expertise from people setting up these brains. i would hope they know `ip` haha